db/blob/blob_stats_record.cc

```diff
@@ -0,0 +1,37 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "db/blob/blob_stats_record.h"
```

Comment:
Just general code style question, should for example, `blob_stats.h`, `blob_stats_record.h` and  `blob_stats_collection.h` be combined into one file? And the same for *.cc files.

I have the impression that our codebase has lots of related structures defined in one file.

Reply:
I don't think we have a hard and fast rule for this; I personally prefer to have separate source files because they can reduce dependencies. (For example, `blob_table_properties_collector.h` only includes `blob_stats.h` but not the other two.) As for the pre-existing stuff, there has been some cleanup work done recently to split up large source files to make file sizes more manageable.